ct_optcon/include/ct/optcon/solver/lqp/HPIPMInterface-impl.hpp

```diff
@@ -630,8 +625,8 @@ void HPIPMInterface<STATE_DIM, CONTROL_DIM>::d_print_e_tran_mat(int row, int col
 }
 
 
-template <int STATE_DIM, int CONTROL_DIM>
-const ct::core::ControlVectorArray<CONTROL_DIM>& HPIPMInterface<STATE_DIM, CONTROL_DIM>::get_lv()
+template <typename MANIFOLD, int CONTROL_DIM>
+const ct::core::ControlVectorArray<CONTROL_DIM>& HPIPMInterface<MANIFOLD, CONTROL_DIM>::get_lv()
```

Comment:
TODO: throw an error at compile time if HPIPM is attempted to be built with a non-euclidean manifold